message-hub/README.md

```diff
@@ -0,0 +1,174 @@
+# Using IBM Message Hub and Apache Kafka to copy documents from Cloudant to Db2 Warehouse on Cloud
+
+This tutorial will use Apache Kafka Connect to publish messages from a Cloudant
+database to an IBM Cloud Message Hub topic. The messages will then be consumed by
+Apache Kafka Connect's JDBC connector to write to a Db2 Warehouse on Cloud table.
+
```

Comment:
Should we add a ToC similar to the other tutorials?

Reply:
Done in 8f427dd61e5c4025362475c422b517c5ce28338b